### PR TITLE
Add jump and aim emotes

### PR DESCRIPTION
**Pull request by cocosolos**
**_I affirm:_**
- [x] that I agree to Topaz Next's [Limited Contributor License Agreement](https://github.com/topaz-next/topaz/blob/release/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have read the [Contributing Guide](https://github.com/topaz-next/topaz/blob/release/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/topaz-next/topaz/blob/release/CODE_OF_CONDUCT.md)
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: src/map/packet_system.cpp [hunk 1]
```diff
@@ -6742,6 +6743,42 @@ void SmallPacket0x118(map_session_data_t* const PSession, CCharEntity* const PCh
...
+        PChar->SetLocalVar("LastEmoteTime", (uint32)timeNowSeconds.time_since_epoch().count());
+    }
+    else
+    {
+        ShowWarning(CL_YELLOW "SmallPacket0x11D: Rate limiting jump packet for %s\n" CL_RESET, PChar->GetName());
+        return;
+    }
+
+    const auto TargetID    = data.ref<uint32>(0x04);
+    const auto TargetIndex = data.ref<uint16>(0x08);
```
Comment (zach2good) [inline src/map/packet_system.cpp]: `targetID`
`targetIndex`
please 🙏

File: src/map/packets/char_emotion.cpp [hunk 2]
```diff
@@ -47,6 +47,26 @@ CCharEmotionPacket::CCharEmotionPacket(CCharEntity* PChar, uint32 TargetID, uint
             ref<uint16>(0x12) = PWeapon->getID();
         }
     }
+    else if (EmoteID == Emote::AIM)
+    {
+        ref<uint16>(0x12) = 65535;
+        CItemWeapon* PWeapon = static_cast<CItemWeapon*>(PChar->getStorage(PChar->equipLoc[SLOT_RANGED])->GetItem(PChar->equip[SLOT_RANGED]));
+        if (PWeapon && PWeapon->getID() != 65535)
+        {
+            if (PWeapon->getSkillType() == 27) // throwing
```
Comment (zach2good) [inline src/map/packets/char_emotion.cpp]: Do we have enums for the skilltypes anywhere?

File: src/map/packets/char_jump.cpp [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+#include "char_jump.h"
+#include "../entities/charentity.h"
+
+CCharJumpPacket::CCharJumpPacket(CCharEntity* PChar, uint16 TargetIndex, uint16 extra)
+{
+    this->id(0x11E);
+    this->length(8);
+
+    ref<uint16>(0x04) = TargetIndex;
```
Comment (zach2good) [inline src/map/packets/char_jump.cpp]: `targetIndex`
please 🙏

**Review by zach2good — APPROVED** [review]
Nice and tidy, just a couple of style dings to look at before merge 👍

**Comment by Xaver-DaRed** [comment]
Does "char_Jump" have to be its own independent packet/file?
and if it has to, since its an emote, maybe calling it "emote_jump" would be better?

**Pull request merged**